src/Kunstmaan/MediaBundle/Validator/Constraints/HasGuessableExtensionValidator.php

```diff
@@ -42,10 +48,10 @@ public function validate($value, Constraint $constraint)
             return;
         }
 
-        $contentType = $this->mimeTypeGuesser->guess($value->getPathname());
+        $contentType = $this->mimeTypeGuesser->guessMimeType($value->getPathname());
```

Comment:
I think this check should be moved to a private service where we check if `$this->mimeTypeGuesser` is not null (meaning the setter was called) and we should use the old class otherwise we can use the new mimetype class. Same for the call bellow